ygen/protogen.go

```diff
@@ -512,7 +544,8 @@ func genProtoEnum(field *yang.Entry) (*protoMsgEnum, error) {
 			continue
 		}
 		// We always add one to the value that is returned to ensure that
-		// we never redefine value 0.
+		// we never redefine value 0. We always specify the names as all upper-case
+		// to ensure that we follow the protobuf style guide.
```

Comment:
We always specify the names as all upper-case to ensure that we follow the protobuf style guide. ->
"Names are converted to upper case to follow the protobuf style guide."
Must be a quiet day since I've gone back to shortening your comments ;-)

Reply:
Done :-), I'm fine with the suggestions ✏️ .